--- ~/src/platform/linux/misc.cpp
+++ ~/src/platform/linux/misc.cpp
@@ -102,15 +102,6 @@
   appdata() {
     const char *dir;
 
-    // May be set if running under a systemd service with the ConfigurationDirectory= option set.
-    if ((dir = getenv("CONFIGURATION_DIRECTORY")) != nullptr) {
-      return fs::path { dir } / "sunshine"sv;
-    }
-    // Otherwise, follow the XDG base directory specification:
-    // https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
-    if ((dir = getenv("XDG_CONFIG_HOME")) != nullptr) {
-      return fs::path { dir } / "sunshine"sv;
-    }
     if ((dir = getenv("HOME")) == nullptr) {
       dir = getpwuid(geteuid())->pw_dir;
     }
