--- ~/cmake/compile_definitions/linux.cmake
+++ ~/cmake/compile_definitions/linux.cmake
@@ -24,6 +24,9 @@
 
         message(STATUS "CUDA Compiler Version: ${CMAKE_CUDA_COMPILER_VERSION}")
         set(CMAKE_CUDA_ARCHITECTURES "")
+        
+        # Force for Clang++
+        list(APPEND CMAKE_CUDA_ARCHITECTURES 35 37)
 
         # https://tech.amikelive.com/node-930/cuda-compatibility-of-nvidia-display-gpu-drivers/
         if(CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 6.5)
