--- ~/cmake/compile_definitions/linux.cmake
+++ ~/cmake/compile_definitions/linux.cmake
@@ -76,7 +76,7 @@
             # set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} -gencode arch=compute_90,code=sm_90")
         endif()
 
-        if(CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 12.0)
+        if(CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 12.0 OR CMAKE_CUDA_COMPILER_ID STREQUAL "Clang")
             list(APPEND CMAKE_CUDA_ARCHITECTURES 35)
             # set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} -gencode arch=compute_35,code=sm_35")
         endif()
